README.md

```diff
@@ -278,7 +425,7 @@ This needs to be done in the Info.plist directly:
 </array>
 ```
 
-#### Step 3: Implement handling deep links in your app
+###### Step 3: Implement handling deep links in your app
 
 Steps 1 & 2 set your app up for receiving deep links but now is when you need to figure out how to handle them within your app.
```

Comment:
This wording feels strange to me. It feels like we're saying "you need to figure out how to handle deep links, but we're not going to tell you. Good luck!"

How about "Steps 1 and 2 enable your app to receive deep links, but you also need to handle these links within your app. Follow the relevant steps below depending on whether you are using UIKit or SwiftUI."

Reply:
LOL does sound that way! Took your suggestion but made a slight change to avoid some redundancy.


###### Step 3: Implement handling deep links in your app

Steps 1 and 2 enable your app to receive deep links, but you also need to handle these links within your app. 
This is done by implementing the [`application:openURL:options:`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application) 
method in your app delegate.

Example:

```swift
func application(
    _ app: UIApplication,
    open url: URL,
    options: [UIApplication.OpenURLOptionsKey : Any] = [:]
) -> Bool {
    guard let components = NSURLComponents(url: url, resolvingAgainstBaseURL: true)
    else {
       print("Invalid deep linking URL")
       return false
    }

    print("components: \(components.debugDescription)")

    return true
}
```

If you are using SwiftUI, then you can implement [`onOpenURL(perform:)`](<https://developer.apple.com/documentation/swiftui/view/onopenurl(perform:)>) as a view modifier in the view you intent to handle deep links. This may or may not be the root of your scene.

Example:

```swift
@main
struct MyApplication: App {
  var body: some Scene {
    WindowGroup {
      ContentView()
        .onOpenURL { url in
          // handle the URL that must be opened
        }
    }
  }
}
```

Finally, we have an example app (`Examples/KlaviyoSwiftExamples`) in the SDK repo that you can reference to get an example of how to implement deep links in your app.

Once the above steps are complete, you can send push notifications from the Klaviyo Push editor within the Klaviyo website. 
Here you can build and send a push notification through Klaviyo to make sure that the URL shows up in the handler you implemented in Step 3.

Additionally, you can also locally trigger a deep link to make sure your code is working using the below command in the terminal.

`xcrun simctl openurl booted {your_URL_here}`